pkg/hintrunner/zero/zerohint_others.go

```diff
@@ -34,8 +36,12 @@ func newMemContinueHint(continueTarget hinter.ResOperander, memset bool) hinter.
 				return err
 			}
 
-			newN := new(f.Element)
-			newN = newN.Sub(n.(*f.Element), &utils.FeltOne)
+			newN, ok := n.(fp.Element)
+			if !ok {
+				return fmt.Errorf("casting n into a felt failed")
+			}
+
```

Comment:
i think its more straightforward to do :

```
newN, ok := n.(fp.Element)
	if !ok {
		return fmt.Errorf("casting n into a felt failed")
	}

newN.Sub(&newN, &utils.FeltOne)
```

Reply:
Right, Modified it to that.